### ISSUE TITLE
Not compatible with Play 2.7

### ISSUE THREAD

**Issue by olivierlemasle** [mohiva/play-silhouette#550]
With Silhouette 5.0.6 and Play 2.7.0-RC3:

```
java.lang.AbstractMethodError: Method com/mohiva/play/silhouette/api/actions/DefaultSecuredErrorHandler.play$api$http$Status$_setter_$REQUEST_HEADER_FIELDS_TOO_LARGE_$eq(I)V is abstract
	at com.mohiva.play.silhouette.api.actions.DefaultSecuredErrorHandler.play$api$http$Status$_setter_$REQUEST_HEADER_FIELDS_TOO_LARGE_$eq(SecuredAction.scala)
	at play.api.http.Status.$init$(StandardValues.scala:221)
	at com.mohiva.play.silhouette.api.actions.DefaultSecuredErrorHandler.<init>(SecuredAction.scala:299)
	at com.mohiva.play.silhouette.api.actions.SecuredErrorHandlerComponents.securedErrorHandler(SecuredAction.scala:363)
	at com.mohiva.play.silhouette.api.actions.SecuredErrorHandlerComponents.securedErrorHandler$(SecuredAction.scala:363)
```
This comes from new status having been added to the trait `play.api.http.Status` (in [StandardValues](https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/http/StandardValues.scala)).

**Comment by akkie** [comment]
The next version of Silhouette will be based on https://github.com/minutemen/silhouette

It will need some time to finalize it and to adapt the documentation.

**Comment by cheleb** [comment]
AFAIU it was an issue with "org.webjars" %% "webjars-play" ... 
2.7.0-RC3 fixes the issue.

**Comment by gguan** [comment]
Play 2.7 is released today. Any ETA about support for Play 2.7 release?

**Comment by dejvid** [comment]
Is it possible to help you with that?

**Comment by akkie** [comment]
@dejvid Yes, you can help with https://github.com/mohiva/play-silhouette/pull/553

**Comment by akkie** [comment]
I've released a snapshot version. Please report any issues back. I'll release a RC1 in a week if no issues will be reported.

**Comment by akkie** [comment]
I've released a RC1

**Comment by manuchadha1979** [comment]
Thanks. I am not sure if this is the right forum to ask this. I upgraded to Play 2.7 and Silhouette 6. At first I got the error

Error:(24, 7) class PasswordRepository needs to be abstract, since value classTag in trait DelegableAuthInfoDAO of type scala.reflect.ClassTag[com.mohiva.play.silhouette.api.util.PasswordInfo] is not defined
class PasswordRepository(utilities:HelperMethods,userRepo:UsersRepository,messagesApi: MessagesApi,langs:Langs) extends DelegableAuthInfoDAO[PasswordInfo]{

Then I tried to define the value as follows

class PasswordRepository(utilities:HelperMethods,userRepo:UsersRepository,messagesApi: MessagesApi,langs:Langs) extends DelegableAuthInfoDAO[PasswordInfo]{

override val classTag = ClassTag[PasswordInfo](classOf[PasswordInfo])

Is this the right way to do things?

abstract val classTag: ClassTag[T]
The class tag for the type parameter.
